src/Pages/ComparisonGraph.cshtml.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Security.Claims;
+using ePiggyWeb.DataManagement;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ePiggyWeb.Pages
+{
+    [Authorize]
+    public class ComparisonGraphModel : PageModel
+    {
+        public decimal Income { get; set; }
+        public decimal Expenses { get; set; }
+        private int UserId { get; set; }
+        [BindProperty]
+        public DateTime StartDate { get; set; }
+        [BindProperty]
+        public DateTime EndDate { get; set; }
+        public void OnGet()
+        {
+            var today = DateTime.Now;
+            StartDate = new DateTime(today.Year, today.Month, 1);
+            EndDate = DateTime.Today;
```

Comment:
Maybe it would be better to see entries until the end of the month? I guess this is subjective though

Reply:
I just took _Swedbank_ filtering by date system as an example, and they don't even allow to go to the future :D, but I just wanted to set from 1st of Month till today period as preset value. And if user wants to see data until the end of the month, they can set that in filter.

Reply:
Yeah, that makes sense